style.css

```diff
@@ -30,69 +19,82 @@ body > div {
 h1 {
     font-size: 24px;
     font-weight: 950;
-    color: #F9FAF8;
-    margin-right: auto;
```

Comment:
No longer needed to space the logo and nav items apart, instead the container uses `justify-content: space-between` to push its flex children apart.